src/main/java/com/iexec/worker/api/ResultProxyApiClient.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.worker.api;
+
+import com.iexec.common.result.ResultModel;
+import feign.Headers;
+import feign.Param;
+import feign.RequestLine;
+
+
+public interface ResultProxyApiClient {
+
+    @Headers("Authorization: {authorization}")
+    @RequestLine("POST /v1/results")
```

Comment:
I have checked in other modules/clients.
We have a kind of tait convention where the `@Header` is defined after the `@RequestLine`
```suggestion
    @RequestLine("POST /v1/results")
    @Headers("Authorization: {authorization}")
```